spec/services/pull_request/parse_repository_spec.rb

```diff
@@ -0,0 +1,12 @@
+describe 'PullRequest::ParseRepository' do
+  context 'within given repo url' do
+    let(:repo_url) { 'https://github.com/example_company/example_project/pull/99' }
+
+    it 'should parse repo name and issue number' do
+      action = PullRequest::ParseRepository.new(repo_url)
+      action.call
+      expect(action.repo_name).to eq('example_company/example_project')
+      expect(action.issue_number).to eq('99')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.